requirements.txt

```diff
@@ -1,5 +1,5 @@
-dask==2022.7.1
-distributed==2022.7.1
+dask>=2022.7.1
+distributed>=2022.7.1
```

Comment:
Should this be `>=2022.8` now that the August release is out? Or do we
not mind because the version resolution will pick the latest version
given the constraint?

Reply:
No, we only pin versions around code freeze time. During development we don't need to enforce a specific minimum version unless it's because some old version is problematic, given that's not the case here, it's useful to be able to go back if needed.